src/com/severalcircles/colors/commands/CommandPersona.java

```diff
@@ -5,13 +5,19 @@
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandExecutor;
 import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
 public class CommandPersona implements CommandExecutor {
     @Override
     public boolean onCommand(CommandSender commandSender, Command command, String s, String[] args) {
+        Player player = (Player) commandSender;
         int chatspam = 0;
-        while (chatspam<3000) {
-            Bukkit.broadcastMessage(ChatColor.DARK_RED + "Joker Persona");
-            chatspam++;
+        if (player.hasPermission("com.severalcircles.colors.persona")) {
+            while (chatspam < 3000) {
```

Comment:
We could reduce this number a bit, the messages fly by fast enough that it just doesn't matter.